tslearn/metrics.py

```diff
@@ -198,23 +136,30 @@ def dtw_path(s1, s2, global_constraint=None,
     ----------
     s1
         A time series.
+
     s2
         Another time series.
         If not given, self-similarity of dataset1 is returned.
+
     global_constraint : {"itakura", "sakoe_chiba"} or None (default: None)
         Global constraint to restrict admissible paths for DTW.
-    sakoe_chiba_radius : int (default: 1)
-        Radius to be used for Sakoe-Chiba band global constraint. Used only if
-        ``global_constraint="sakoe_chiba"``.
-    itakura_max_slope : float (default: 2.)
-        Maximum slope for the Itakura parallelogram constraint. Used only if
-        ``global_constraint="itakura"``.
+
+    sakoe_chiba_radius : int or None (default: None)
```

Comment:
I would update the docs and explicitly say that itakura_max_slope cannot be set if this one is set, and vice versa.

Reply:
Done (in every place in `metrics.py` that have these parameters).